ci/builders/mac_ios_engine.json

```diff
@@ -4,7 +4,8 @@
             "drone_dimensions": [
                 "device_type=none",
                 "mac_model=Macmini8,1",
-                "os=Mac-12"
+                "os=Mac-12",
```

Comment:
Some of these have `"mac_model=Macmini8,1"` and some don't. Is that important?

Reply:
Good point mac_model=Macmini8,1 will resolve to x86. Removing them for consistency.